testing/coreruleset/coreruleset_test.go

```diff
@@ -120,20 +131,113 @@ func BenchmarkCRSSimplePOST(b *testing.B) {
 	}
 }
 
-func crsWAF(t testing.TB) *corazawaf.WAF {
-	t.Helper()
+func TestFTW(t *testing.T) {
 	files := []string{
-		"../coraza.conf-recommended",
-		path.Join(crspath, "crs-setup.conf.example"),
-		path.Join(crspath, "rules/", "*.conf"),
+		filepath.Join("..", "..", "coraza.conf-recommended"),
+		filepath.Join(crspath, "crs-setup.conf.example"),
+		filepath.Join(crspath, "rules", "*.conf"),
 	}
-	waf := corazawaf.NewWAF()
-	parser := seclang.NewParser(waf)
+	conf := coraza.NewWAFConfig()
+
+	conf = conf.WithDirectives(`
+SecAction "id:900005,\
+  phase:1,\
+  nolog,\
+  pass,\
+  ctl:ruleEngine=DetectionOnly,\
+  ctl:ruleRemoveById=910000,\
+  # Interferes with ftw log scanning
```

Comment:
I've noticed this annoying log when running FTW on coraza-proxy-wasm but somehow there FTW still worked without disabling this but not here, not sure what the difference is. /cc @M4tteoP

Reply:
And oh #458 (not related to FTW behavior difference but will fix the root cause)